### PR TITLE
[updatecli] Update custom-distribution-service appVersion version to custom-distribution-service-0.0.2

### PR DESCRIPTION
**Pull request by olblak**
## Changelog

Release published on the 2020-09-23 17:25:49 +0000 UTC at the url https://github.com/jenkinsci/custom-distribution-service/releases/tag/custom-distribution-service-0.0.2

* Add a single page router that will redirect all missing requests to index.html (#163) @halkeye
* Prepare for 0.1-alpha release (#161) @sladyn98

## Reports

	Source:
		✔  (githubRelease)

 
 	Condition:
		✔  Docker Image Published on Registry(dockerImage)

## Remark

This pull request was automatically created using [olblak/updatecli](https://github.com/olblak/updatecli).
Please report any issues with this tool [here](https://github.com/olblak/updatecli/issues/new)

### CODE REVIEW

File: charts/custom-distribution-service/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.1-alpha-1
+appVersion: custom-distribution-service-0.0.2
```
Comment (timja) [inline charts/custom-distribution-service/Chart.yaml:2]: i think we should not have the app name in the tag for this

Comment (halkeye) [reply charts/custom-distribution-service/Chart.yaml:2]: blocker for this update?

**Comment by halkeye** [comment]
code owners isn't working :(

File: charts/custom-distribution-service/Chart.yaml (hunk 1 above)
Comment (halkeye) [reply charts/custom-distribution-service/Chart.yaml:2]: addressed for next release https://github.com/jenkinsci/custom-distribution-service/pull/164

**Review by halkeye — APPROVED** [review]

File: charts/custom-distribution-service/Chart.yaml (hunk 1 above)
Comment (timja) [reply charts/custom-distribution-service/Chart.yaml:2]: not a blocker

**Review by halkeye — APPROVED** [review]

**Pull request merged**